components/cloud/azure/src/azblob.rs

```diff
@@ -339,6 +383,26 @@ impl AzureUploader {
         }
     }
 
+    #[inline]
+    fn adjust_put_builder(&self, builder: PutBlockBlobBuilder) -> PutBlockBlobBuilder {
+        // the encryption scope and the access tier can not be both in the HTTP headers
+        if let Some(scope) = &self.encryption_scope {
+            return builder.encryption_scope(scope.deref().clone());
+        }
+
+        // the encryption customer provided key and the access tier can not be both in
+        // the HTTP headers
+        if let Some(key) = &self.encryption_customer {
+            return builder.encryption_key::<(String, String)>(key.clone().into());
```

Comment:
So, it is impossible to provide both encryption scope and encryption key? Is this excepted?

Reply:
Yes, when both the encryption scope and the encryption key are in the headers, the azure server will return an error.